<!DOCTYPE html>
<html lang="en">

<head>

  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="widtd=device-widtd, initial-scale=1">
  <meta name="description" content="">
  <meta name="autdor" content="">

  <title>Washington University - Penn State IGEM</title>

  <!-- Bootstrap Core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom CSS -->
  <link href="css/igem.css" rel="stylesheet">

  <!-- Custom Fonts -->
  <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
  <link href='http://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view tde page via file:// -->
  <!--[if lt IE 9]>
  <script src="http://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
  <script src="http://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->

  <!-- Background Req -->
  <script src="js/underscore-min.js"></script>
  <script src="js/backbone-min.js"></script>
  <script src="js/d3.min.js" charset="utf-8"></script>
  <script src="js/module-manager.js"></script>


</head>

<body id="page-top" class="index">

  <!-- Navigation -->
  <nav class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header page-scroll">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand page-scroll" href="#page-top">Nitrogenius</a>
      </div>

      <!-- Collect tde nav links, forms, and otder content for toggling -->
      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
        <ul class="nav navbar-nav navbar-right">
          <li class="hidden">
            <a href="#page-top"></a>
          </li>
          <li>
            <a class="page-scroll" href="#selection">Selection</a>
          </li>
          <li>
            <a class="page-scroll" href="#design">Design</a>
          </li>
          <li>
            <a class="page-scroll" href="#cloning">Cloning</a>
          </li>
          <li>
            <a href="mailto:ayekedavidr@wustl.edu?subject=Bug">Report Bugs</a>
          </li>
        </ul>
      </div>
      <!-- /.navbar-collapse -->
    </div>
    <!-- /.container-fluid -->
  </nav>

  <!-- Header -->
  <header>
    <div class="container">
      <div class="intro-text">
        <div class="intro-lead-in">Check out our Team!</div>
        <div class="intro-heading"></div>
      </div>
    </div>
  </header>

  <!-- Background Canvas -->
  <div id = "background-canvas"></div>



  <!-- Project Overview -->
  <!-- Main container -->
  <div class="container">
    <section id="selection" class="bg-light-gray row sectionNum1">
      <h2>Selecting Genes</h2>
      <p>In designing our minimal nif cluster plasmids, we considered each of tde 23 genes on tde PSL2397 nif plasmid, derived from Cyanotdece 51142, and selected 14 tdat we believe may be essential for nitrogenase activity. </p>
      <p>We used literature on tde creation of otder minimal nif clusters and took into account a few factors: </p>
      <ul>
        <li>function of tde gene, if established (**find out source on Cheryl’s nif chart)</li>
        <li>homology witd genes in otder minimal nif clusters</li>
        <li>homology witd genes in E. Coli’s genome</li>
      </ul>
      <table class="table">
        <tr>
          <th>Gene</th>
          <th>Function</th>
          <th>Justification</th>
        </tr>
        <tr>
          <td>nifH</td>
          <td>Nitrogenase iron protein</td>
          <td>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</td>
        </tr>
        <tr>
          <td>nifD</td>
          <td>Nitrogenase molybdenum-iron protein alpha chain</td>
          <td>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</td>
        </tr>
        <tr>
          <td>nifK</td>
          <td>Nitrogenase molybdenum-iron protein beta chain</td>
          <td>Main structural component, can't have activity without all necessary structural components. Additionally, all literature looked at expressed nifH, nifD, nifK at high levels</td>
        </tr>
        <tr>
          <td>nifE</td>
          <td>Nitrogenase Fe-Mo cofactor and assembly protein</td>
          <td>Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.</td>
        </tr>
        <tr>
          <td>nifN</td>
          <td>Nitrogenase Fe-Mo cofactor and assembly protein</td>
          <td>Seems essential for enzymatic assembly. All literature read expressed nifE, nifN at variable levels.</td>
        </tr>
        <tr>
          <td>nifB</td>
          <td>Homocitrate syntdase/FeMo-co core syntdesis</td>
          <td>Voigt paper claims nifB is a key component in putting together the FeMo-co core. Nif B was expressed at high levels</td>
        </tr>
        <tr>
          <td>nifU</td>
          <td>Encodes proteins for early Fe-S formation; proteins for component maturation</td>
          <td>Voigt expressed nifU at low levels. It could be a necessary component in formation of Fe-S clusters </td>
        </tr>
        <tr>
          <td>nifS</td>
          <td>Cysteine desulfurase; used in MoFe cluster syntdesis</td>
          <td>Ludden speculates that cysteine desulfurase plays a key role in Fe-S cluster assembly</td>
        </tr>
        <tr>
          <td>nifV</td>
          <td>Homocitrate syntdase</td>
          <td>Included by Voigt in his work, but expressed at low levels. Nitrogenase mutants which didn't contain homocitrate had problems with substrate association</td>
        </tr>
        <tr>
          <td>nifW</td>
          <td>Nitrogenase-stabilizing/protective protein</td>
          <td>Also included by Voigt in his work, low expression. It could be a necessary component of nitrogen fixation</td>
        </tr>
        <tr>
          <td>nifZ</td>
          <td>Encodes proteins for Fe-S formation; proteins for component maturation</td>
          <td>Also included by Voigt in his work, low expression. Helps maintain enzyme's structural integrity</td>
        </tr>
        <tr>
          <td>hesA</td>
          <td>Related to molybdenum cofactor; part of tdiF_MoeB_HesA protein family which are involved in molybdopterin and tdiamine biosyntdesis; E. coli has botd tdiF and MoeB</td>
          <td>May have role in creating Molybdenum cofactor, a necessary component to the FeMo-co core. </td>
        </tr>
        <tr>
          <td>hesB</td>
          <td>Fe-S cluster biosyntdesis, necessary for enzyme syntdesis</td>
          <td>Fe-S clusters are important components of the Fe protein and the FeMo protein. Closest match within E. coli had 49% identity. To ensure correct metallocluster synthesis, we need to use this gene.</td>
        </tr>
        <tr>
          <td>cysE2</td>
          <td>serine O-acetyltransferase</td>
          <td>Serine O-acetyltransferases are important enzymes in cysteine metabolism. Since the formation of FeS clusters is predicated on cysteine reactions, presence of this enzyme may help increase synthetic efficiency</td>
        </tr>
      </table>
      <p>Our analysis left us with 14 genes to be included in the minimal nif plasmids. We also designed a 15-nucleotide ribosome binding site to insert directly upstream of each gene. </p>
      <p>RBS sequences were designed by the Penn State Salis Lab's <a href="https://salislab.net/software/">RBS Calculator</a>.Each RBS was designed to have a Translation Initiation Rate close to 50,000 au (Arbitrary Units) as descrbibed in considerations 1.2a &amp; 1.2b of the Salis <a  href="#" data-container="body" data-toggle="popover" data-placement="top" data-content="Howard, Salis. The ribosome binding site calculator." >paper</a>. </p>
    </section>
    <section id= "design">
      <h2>Plasmid Design</h2>
      <p>The literature also indicated that the expression levels of each gene need to be finely tuned in order to achieve optimal nitrogenase activity. We decided to use 2 different inducible promoters to give us more control over expression levels, and grouped the genes into ones we thought would need to be expressed at higher versus lower levels. </p>
      <p>We decided to create 2 plasmids with 7 genes each rather than one plasmid with 14 genes, which would simplify cloning greatly. We designed one plasmid containing genes that we generally believed needed to be expressed at higher levels, including the enzyme structural genes. </p>
      <p>In ordering the genes on the plasmids, we mimicked gene orders we observed in the natural Cyanothece cluster and in other minimal clusters in the literature. Since we did not have the resources to codon optimize, we also considered codon usage in placement of genes on the plasmids. One gene, cysE2, has particularly bad codon usage for E. Coli, so we placed it directly downstream of the promoter. </p>
    </section>
    <section id = "cloning">
      <h2>Primer Design and Cloning Strategy</h2>
      <p>We designed primers for use with Golden Gate cloning to create the plasmids. Our general strategy for primer design was to add a BsaI site and one intervening base before the fifteen-base RBS before the annealing portion on the forward primer. When possible, we used the first four bases of the RBS as the overhang for ligation, rather than adding in bases and creating a scar. On the reverse primer, we added a BsaI site and one intervening base before adding the reverse complement of the overhang that it should match up with. In designing primers, we considered:</p>
      <ul>
        <li>maintaining a primer length of 60 nucleotides or less</li>
        <li>having a melting temperature (T m) of the primer-template interaction between about 58 and 62 degrees Celsius</li>
        <li>matching the melting temperature of primers in a primer pair as closely as possible to each other</li>
        <li>maintaining a hairpin melting temperature as low as possible, and always less than the T m of the primer-template interaction</li>
        <li>maintaining heterodimer and homodimer interaction delta G values to be less in magnitude than -15 kcal/mol</li>
        <li>adding up to 6 adenines before the BsaI site where possible</li>
        <li>having overlap sequences that are distinct enough from other overlap sequences used for the same plasmid</li>
          <ul>
            <li>our criterion was to have a maximum of 2 of the same base in the same position. i.e. GACT and CAGT are acceptable as only A and T fall in the same positions. However, GACT and CACT are too similar and one must be changed to avoid improper ligation</li>
          </ul>
      </ul>
      <p>In order to balance all of these constraints, a couple of changes to the coding sequence of the genes were made; however, degenerate codons were used so that the amino acid sequence was not disrupted. </p>
    </section>

    <section id="team"  class="bg-light-gray row sectionNum1 sectionNum2 sectionNum3 sectionNum4 sectionNum5 sectionNum6 ">
      <div class="container">
        Under construction
      </div>
    </section>

  </div>

  <footer  class="bg-white row">
    <div class="container">
      <!-- sponsors Aside -->
      <section id="sponsors" class="row  sectionNum5">
        <div class="container">
          <div class="row">
            <h2>Sponsors</h2>
          </div>

          <div class="row">
            <div class="col-md-4 col-sm-6">

              <div id="WashULogo" class="logo"></div>
            </div>
            <div class="col-md-4 col-sm-6">
              <div id="ICaresLogo" class="logo"></div>
            </div>
            <div class="col-md-4 col-sm-6">
              <div id="WashUArtLogo" class="logo"></div>
            </div>
            <div class="col-md-4 col-sm-6">
              <div id="SnapGeneLogo" class="logo"></div>
            </div>
            <div class="col-md-4 col-sm-6">
              <div id="Monsanto" class="logo"></div>
            </div>
            <div class="col-md-4 col-sm-6">
              <div id="SigmaAldrichLogo" class="logo"></div>
            </div>
          </div>
        </div>
      </section>
      <div class="row">
        <div class="col-md-4">
          <span class="copyright">Copyright &copy; Washington University in St. Louis Igem 2015</span>
        </div>
        <div class="col-md-4">
          <ul class="list-inline social-buttons">
            <li><a href="http://twitter.com/WashUiGEM"><i class="fa fa-twitter"></i></a>
            </li>
            <li><a href="http://www.facebook.com/WashuIgem"><i class="fa fa-facebook"></i></a>
            </li>
            <li><a href="http://www.youtube.com/channel/UCjO_tMiJdx6hCyCREsMBYUQ"><i class="fa fa-youtube"></i></a>
            </li>
          </ul>
        </div>

      </div>
    </div>
  </footer>

  <!-- jQuery -->
  <script src="js/jquery.js"></script>

  <!-- Bootstrap Core JavaScript -->
  <script src="js/bootstrap.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>
  <script src="js/classie.js"></script>
  <script src="js/cbpAnimatedHeader.js"></script>

  <!-- Contact Form JavaScript -->
  <script src="js/jqBootstrapValidation.js"></script>
  <script src="js/contact_me.js"></script>

  <!-- Footnotes -->
  <script src="js/bootstrap.footnotes.css"></script>

  <!-- Custom tdeme JavaScript -->
  <script src="js/agency.js"></script>



  <!-- Background Script -->
  <script src="js/background.js"></script>

  <!-- Inline script -->
  <script>
  // Activate Footnotes
  $.footnotes();
  </script>
</body>

</html>
